hot-reloading: Add descriptions for Windows (#1209)
fluent/fluent-bit-docs: administration/hot-reload.md

$ curl -X POST -d {} localhost:2020/api/v2/reload
```

> For Windows, to represent empty body, users have to use `'{}'` as a body instead of raw `{}`:

Reviewer: ```suggestion
On Windows, use `'{}'` to represent an empty body instead of raw `{}`:
```

Reviewer: To be fair you could do this on Linux too I think?

Author: No, it's Windows specific.

Reviewer: No, I mean you can quote it on Linux as well. It works fine so maybe we should just use the one format for both?

Both work for Linux:
```shell
$ curl -X POST -d {} 127.0.0.1:2020/api/v2/reload
{"reload":"done","status":0}
$ curl -X POST -d '{}' 127.0.0.1:2020/api/v2/reload
{"reload":"done","status":0}
```

Author: Ah, OK. We can do that. Let's move to use quoted empty body.
